### PR TITLE
feat: added dark mode for inline banners

### PR DESCRIPTION
**Pull request by queen-x-vee**
## What does this PR do?
This PR adds the feature of dark mode for the inline banners. I created a booking appearance file where I gave a preview if the different colours of the inline banners according to their variants. I also modified the the top banner components and show toast components
if a user device is in dark mode, then the colours of the inline banners and toast notifications will correspond to dark mode colours. But if the user device is in light mode then it will be in light mode colours

Fixes #9467

 Loom Video: https://www.loom.com/share/843dad9b6f7b41acb7b40003766f1410

**Environment**: Staging(main branch) / Production

## Type of change

- New feature (non-breaking change which adds functionality)

## How should this be tested?

set your device theme to dark them and observe the changes on the appearance page in settings, or from toast notifications, then set to light theme and see corresponding colours

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -7,5 +7,13 @@
...
-  "tailwindCSS.experimental.classRegex": [["cva\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"]]
+  "tailwindCSS.experimental.classRegex": [
+    [
+      "cva\\(([^)]*)\\)",
+      "[\"'`]([^\"'`]*).*?[\"'`]"
+    ]
+  ],
+  "cSpell.words": [
+    "calcom"
+  ]
```
Comment (Thiagomn1) [inline .vscode/settings.json:18]: Can you explain why is this change required?

File: packages/ui/components/toast/showToast.tsx [hunk 2]
```diff
@@ -24,11 +24,11 @@ export const SuccessToast = ({ message, toastVisible }: IToast) => (
 export const ErrorToast = ({ message, toastVisible }: IToast) => (
   <div
     className={classNames(
-      "animate-fade-in-up bg-error text-error mb-2 flex h-auto items-center space-x-2 rounded-md p-3 text-sm font-semibold shadow-md rtl:space-x-reverse md:max-w-sm",
+      "animate-fade-in-up mb-2 flex h-auto items-center space-x-2 rounded-md  bg-[#F9E3E2] p-3 text-sm font-semibold text-[#752522]  shadow-md rtl:space-x-reverse dark:bg-[#752522]  dark:text-[#F8D8B0]  md:max-w-sm",
```
Comment (Thiagomn1) [inline packages/ui/components/toast/showToast.tsx:27]: We should avoid using inline hardcoded values and use the theme colors available in tailwind config 
i.e: F9E3E2 is available as bg-error or text-error

File: .vscode/settings.json (hunk 1 above)
Comment (queen-x-vee) [reply .vscode/settings.json:18]: Hello Thiagomn, I just realized that these changes are from my local vscode settings, I should not have push it. I will revert it

File: packages/ui/components/toast/showToast.tsx (hunk 2 above)
Comment (queen-x-vee) [reply packages/ui/components/toast/showToast.tsx:27]: Oh thanks for pointing that out, I will update it.

Comment (Toledodev) [reply packages/ui/components/toast/showToast.tsx:27]: Hi @queen-x-vee 😄, calcom have this file with all the custom colors: packages\config\tailwind-preset.js

Comment (queen-x-vee) [reply packages/ui/components/toast/showToast.tsx:27]: Thanks! @Toledodev , I will look at it asap and make the neccessary changes

File: .vscode/settings.json (hunk 1 above)
Comment (queen-x-vee) [reply .vscode/settings.json]: resolved it

File: packages/ui/components/toast/showToast.tsx (hunk 2 above)
Comment (queen-x-vee) [reply packages/ui/components/toast/showToast.tsx]: hi @Thiagomn1 @Toledodev  , I've resolved this

**Review by Toledodev — APPROVED** [review]
Hi @queen-x-vee, sorry for the delay 😄 
This PR looks very nice, great job 👍 🚀

**Comment by queen-x-vee** [comment]
> Hi @queen-x-vee, sorry for the delay 😄
> This PR looks very nice, great job 👍 🚀

Thank you! @Thiagomn1
